dpl-docs/source/app.d

```diff
@@ -16,7 +16,7 @@ int main(string[] args)
 	environment["GIT_TARGET"] = git_target;
 	environment["NO_EXACT_SOURCE_CODE_LINKS"] = noExactSourceCodeLinks ? "1" : "0";
 	setLogFormat(FileLogger.Format.plain);
-	if (args.length > 1 && args[1] != "filter")
-	    args ~= "--html-style=pretty";
+    if (args.length > 1 && args[1] != "filter")
+        args ~= "--html-style=pretty";
```

Comment:
What is this? Looks like a whitespace-only change?

Reply:
Weird. Sorry. Must have gone something wrong when rebasing. Fixed.